runtime/Cargo.toml

```diff
@@ -314,8 +331,10 @@ runtime-benchmarks = [
     'frame-system/runtime-benchmarks',
     'pallet-balances/runtime-benchmarks',
     'pallet-timestamp/runtime-benchmarks',
+    "pallet-collator-selection/runtime-benchmarks",
 ]
 std = [
+    'scale-info/std',
```

Comment:
please use double quotes instead of single quotes for consistency in these Cargo.toml files